source/base/Debug.ooc

```diff
@@ -28,16 +29,21 @@ DebugLevel: enum {
 }
 
 Debug: class {
-	_level: static Int = DebugLevel Everything
+	_level: static DebugLevel = DebugLevel Everything
 	printFunction: static Func (String) = func (s: String) { println(s) }
 	initialize: static func (f: Func (String)) {
 		This printFunction = f
 	}
-	print: static func (printOut: String, level: Int = 1) {
+	print: static func (string: String, level := DebugLevel Everything) {
```

Comment:
Now that you changed it, do we want consistency here? Either `text: Text`/`string: String` or `message: Text`/`message: String`